docs/respondent_management_to_electronic_questionnaire.rst

```diff
@@ -18,29 +22,46 @@ The following metadata keys are always required for the survey runner, they do n
 ``exp``
   JWT Expiration Time claim, see https://tools.ietf.org/html/rfc7519#section-4.1.4
 ``ru_ref``
-  The responding unit reference id - with checkletter appended.
+  The responding unit reference id - with checkletter appended
 ``eq_id``
-  The eQ questionnaire instance id.
+  The eQ questionnaire instance id
 ``collection_exercise_sid``
   A reference number used to represent the collection exercise inside the ONS
 ``form_type``
   The particular form_type for a responding unit
 ``ru_name``
-  The name of the responding unit. Could be a business name or person name. Not required, however at least one of ``ru_name`` and ``trad_as`` **must** be present.
+  The name of the responding unit. Could be a business name or person name. Not required, however at least one of ``ru_name`` and ``trad_as`` **must** be present
 ``trad_as``
-  The trading as name for a responding unit. Temporary until wider refactor. Not required, however at least one of ``ru_name`` and ``trad_as`` **must** be present.
+  The trading as name for a responding unit. Temporary until wider refactor. Not required, however at least one of ``ru_name`` and ``trad_as`` **must** be present
 ``variant_flags``
-  JSON object containing name:value pairs describing variations in the questionnaire e.g. `{"flag_1": true, "flag_2": false}`. The only value type allowed is boolean i.e. `true` or `false`. Even if no flags are passed, an empty object mush be passed.
+  JSON object containing name:value pairs describing variations in the questionnaire e.g. `{"flag_1": true, "flag_2": false}`. The only value type allowed is boolean i.e. `true` or `false`. Even if no flags are passed, an empty object mush be passed
 ``user_id``
   The id assigned by the respondent management system
 ``period_id``
   A numerical reference to either a month or quarter time period
 ``account_service_url``
   The url of the account service (i.e. rrm or ras) used to launch the survey
 
+Optional Fields
+---------------
+The runner can optionally accept the following keys.
+
+``period_str``
+  A display name for the ``period_id`` referenced above
+``language_code``
+  Language code identifier, used to change language displayed. Format as per ISO-639-1 (https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) e.g. "en" for English; "cy" for Welsh. This parameter is currently optional; the default is "en"
+``survey_url``
+  A URL for a remote survey JSON. This claim is used to tell Survey Runner to load the schema JSON from a remote location
+``ru_ref``
```

Comment:
Is this optional? having a look around the code, im not sure if it will handle it not being there?